resources/views/admin/dashboard.blade.php

```diff
@@ -1,25 +1,59 @@
 @extends('layouts.master')
 
 @section('body')
+    <body onload="onLoad()">
```

Comment:
The `<body>` element is already included in the `layouts.master` blade, so there's no need to include it again. There's also no need for the `onload` - any JS that you put in the <script> tags will be executed when the page loads. So essentially, you just need to make sure that you select the proper elements before you try to use them.